changelog/8838.breaking.rst

```diff
@@ -0,0 +1,17 @@
+As per our policy, the following features have been deprecated in the 6.X series and are now
+removed:
+
+* ``pytest._fillfuncargs`` function.
+
+* ``pytest_warning_captured`` hook - use ``pytest_warning_recorded`` instead.
+
+* ``message`` parameter of ``pytest.raises``.
```

Comment:
Just realized I didn't see this removal, did I miss something?

Reply:
Good catch, I copy/pasted from the 6.x PR, left this by mistake.